docs/api-docs/channels/channels-pos-tutorial.md

```diff
@@ -2,28 +2,35 @@
 
 <div class="otp" id="no-index">
 
-### On this page
-- [Creating a channel](#creating-a-channel)
-- [Specifying the platform](#specifying-the-platform)
-- [Configuring UI sections](#configuring-ui-sections)
-- [Rendering UI sections](#rendering-ui-sections)
-- [Related resources](#related-resources)
-
+## On this page

```

Comment:
```suggestion
### On this page

```

Reply:
I changed this to an H2 header so that the headers are ranked in order when the page renders to HTML. As it was formatted before, the document skips from an H1 to an H3. It seems pretty minor, but using headers for styling can throw screen readers off and might be a web accessibility issue (https://www.w3.org/WAI/tutorials/page-structure/headings/)

Reply:
I am good with changing it to H2 to make sure the page renders correctly but I will defer to @uwikaiddi on this.